pkg/api/grpcServer.go

```diff
@@ -155,14 +169,22 @@ func (s *GRPCServer) streamLoop(stream grpc.ServerStream, request proto.Message,
 			return err
 		}
 
+		var span ot.Span
```

Comment:
I'd go with 1) for now. If someone asks, we can then see if adding an integer counter would be useful.